test/README.md

```diff
@@ -5,11 +5,27 @@ This directory contains simple test scripts to check if a Strimzi Operator Kafka
 ## Requirements
 
 - A kubernetes cluster with Kafka deployed via Strimzi Operator
+
+## Liveness Check from Inside the cluster
+### Test Requirements
 - `kubectl` should be configured to point to this cluster
 
-## Liveness Check
 ### Running the test
```

Comment:
@mtarng I think we should call out that these tests only going through one (hardcoded) broker. Perhaps we can have a `TODO` section on this readme to indicate things we want to improve in the future